### PR TITLE
Public services "how-to"

### PR DESCRIPTION
**Pull request by srosam**

### CODE REVIEW

File: docs/platform/how-to/deploy-public-page.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Public services
+
+The Quix SaaS platform allows you to deploy public facing web pages and API's.
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: API's -> APIs.
public facing -> public-facing.
Can we extend the first paragraph to explain what this How-To will help the user achieve.

File: docs/platform/how-to/deploy-public-page.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Public services
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: Should we make the title more active, for example, "Deploying public services" or perhaps "How to deploy a public service"?

File: docs/platform/how-to/deploy-public-page.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+The Quix SaaS platform allows you to deploy public facing web pages and API's.
+
+## Library samples
+
+In our Library you can find our `Web API Template` which demonstrates how to create a API using `Node`.
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md:9]: Should Node be in backticks? Node.js.

File: docs/platform/how-to/deploy-public-page.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## The code
+
+When running locally it is usual to see local addresses like `127.0.0.1` and `localhost`, however, these cannot be used and will result in routing errors.
+
+Please ensure your service is hosted on `0.0.0.0` and uses port `80`.
+
+Examples of this for various languages are:
+
+ - Node JS which can be used to host Dash apps and any Javascript based websites:
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: "Node JS" -> Node.js
Javascript -> JavaScript.
JavaScript-based.

What are Dash apps? Do we need to provide a link?

File: docs/platform/how-to/deploy-public-page.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ - For Streamlit (just set the port in `main.py`)
+    ```sh
+    sys.argv = ["streamlit", "run", "streamlit_file.py", "--server.port=80"]
+    ```
+
+## Deploy a public service
+
+To access your public facing service or web site you must enable `Public Access` at deployment time.
+
+1. Switch on `Public Access`
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: Terminate with period.

File: docs/platform/how-to/deploy-public-page.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+2. If needed adjust the `URL prefix` to suit your needs
+
+![Deployment dialog](../images/deployment-dialog.png){width=450px}
+
+## Security
+
+Please note that the basic examples, included in our library, do not include any security features and come with no warranty.
+
+We advise that you build in a security layer to ensure your data is secure and only the intended recipients have access to it.
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: > We advise that you build

Quix advises you to build...

File: docs/platform/how-to/deploy-public-page.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+Read more about Python API security [here](https://betterprogramming.pub/secure-your-python-api-7e52c475665e){target="_blank"} and [here](https://curity.io/resources/guides/api/python/){target="_blank"}
+
+Fid out more [here](https://auth0.com/blog/complete-guide-to-angular-user-authentication/){target="_blank"}, if you're interested in securing an Angular application.
```
Comment (tbedford) [inline docs/platform/how-to/deploy-public-page.md]: Fid -> Find.

**Review by tbedford — CHANGES_REQUESTED** [review]
Minor comments.

**Comment by SteveRosam** [comment]
@tbedford updated based on your comments.

**Review by tbedford — APPROVED** [review]
LGTM!

**Pull request merged**